nodes.py

```diff
@@ -58,6 +58,37 @@ def INPUT_TYPES(s):
     def combine(self, conditioning_1, conditioning_2):
         return (conditioning_1 + conditioning_2, )
 
+class ConditioningAverage :
+    @classmethod
+    def INPUT_TYPES(s):
+        return {"required": {"conditioning_from": ("CONDITIONING", ), "conditioning_to": ("CONDITIONING", ),
+                              "conditioning_from_strength": ("FLOAT", {"default": 1.0, "min": 0.0, "max": 10.0, "step": 0.1}),
+                              "conditioning_to_strength": ("FLOAT", {"default": 0.0, "min": 0.0, "max": 10.0, "step": 0.1})
```

Comment:
Does it actually make sense to have 2 different strengths? Is there a case where to_strength won't be (1.0 - from_strength) ?

Reply:
you are right and I addressed this in the latest commit. I did do some testing and if the text fields for both conditioning inputs are the same, doesn't matter which order, it will always be the same frame generated. This is in-line with previous behaviour when two strength inputs were present.